pkg/reconciler/stats_reporter_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"testing"
+	"time"
+)
+
+const reconcilerMockName = "MockReconciler"
+
+func TestReporter_ReportDuration(t *testing.T) {
```

Comment:
What about the Report method tests? Also take a look at: https://github.com/knative/pkg/blob/master/controller/stats_reporter_test.go - it has some helper functions that makes writing unit tests for stats reporters easier.

Reply:
It seems like Report isn't being used at all. Please remove the unused methods.

Reply:
Done